### ISSUE TITLE
Reproduce result on UnB-AVC dataset

### ISSUE THREAD

**Issue by hazelAutumn** [charlotte9524/ANNAVQA-pytorch#1]
Hi,
Thanks for your great works. I have tried to reproduce your resutls I kept the default settings of training as in your code. The accuray on the LIVE-SJU was similar to what is reported in your paper. However, the accuracy on the UnB-AVC is low: SROCC=0.4336, KROCC=0.3731, PLCC=0.4649, RMSE=0.1485.

The number of frames for extracting spatial is set to 240 for UnB-AVC (it is 192 for the LIVE-STJU), the number of the selected saliency patches is 25.
Could you provide more information about the experiment setting that you implemented (e.g: learning rate, number of training epochs?)

<img width="542" height="135" alt="Image" src="https://github.com/user-attachments/assets/945d776e-c815-4e5f-964e-d4d2babf32dc" />

**Comment by charlotte9524** [comment]
Since the UnB-AVC dataset is smaller than the LIVE-SJTU dataset, we used a higher learning rate for it. You can find the detailed settings of learning rates and training epochs for both full-reference and no-reference cases in the following paper:
Cao, Y., Min, X., Sun, W. and Zhai, G., 2023. Attention-guided neural networks for full-reference and no-reference audio-visual quality assessment. IEEE Transactions on Image Processing, 32, pp.1882-1896.
If you encounter any other issues during reproduction, please feel free to reach out to me via GitHub or email.

**Comment by hazelAutumn** [comment]
Hi,
Thanks for your quick response. I found the issue, it was caused by reading the video with skivideo. I fixed it and got similar results to those in your paper.
